x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/function/FunctionDefinition.java

```diff
@@ -23,18 +23,22 @@
     private final String name;
     private final List<String> aliases;
     private final Class<? extends Function> clazz;
+    private final Builder builder;
+
     /**
      * Is this a datetime function compatible with {@code EXTRACT}.
      */
+    // TODO: needs refactoring
```

Comment:
comments always welcome!
I'd only add that adding more details might help anyone else picking this up. (but i favour existence of the comment over its completeness.)